azurerm/resource_arm_monitor_activity_log_alert.go

```diff
@@ -75,7 +75,7 @@ func resourceArmMonitorActivityLogAlert() *schema.Resource {
 								"Policy",
 								"Recommendation",
 								"Security",
-								"Service Health",
+								"ServiceHealth",
```

Comment:
could we [also update this in the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/monitor_activity_log_alert.html.markdown#L86)?

Reply:
Done